app/helpers/evaluation_forms_helper.rb

```diff
@@ -1,4 +1,15 @@
 # frozen_string_literal: true
 
 module EvaluationFormsHelper
+  def status_colors
+    {
+      draft: "FireBrick",
+      ready: "DarkGoldenRod",
+      published: "green"
+    }
+  end
```

Comment:
these stauts_colors aren't needed anymore, but if we do they should be matching the figma values, e.g. the green is `#4d8055`